README.adoc

```diff
@@ -20,7 +20,7 @@ The Spring Authorization Server project provides software support through the ht
 https://tanzu.vmware.com/spring-runtime[Commercial support], which offers an extended support period, is also available from VMware.
 
 == Getting Started
-The first place to start is to read the https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-01[OAuth 2.1 Authorization Framework] to gain an in-depth understanding on how to build an Authorization Server.
+The first place to start is to read the https://www.ietf.org/archive/id/draft-ietf-oauth-v2-1-07.html[OAuth 2.1 Authorization Framework] to gain an in-depth understanding on how to build an Authorization Server.
```

Comment:
Please update to https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-07

Reply:
done